### ISSUE TITLE
Need to be able to toggle on "cookie policy" agreement notification

### ISSUE THREAD

**Issue by jamalex** [learningequality/kolibri#8148]
For online hosted instances of Kolibri, for some jurisdictions, we need to be able to toggle on a simple [cookie notice agreement](https://gdpr.eu/cookies/#:~:text=To%20comply%20with%20the%20regulations,cookies%20except%20strictly%20necessary%20cookies.&text=Make%20it%20as%20easy%20for,consent%20in%20the%20first%20place.).

Fortunately, we can keep it fairly simple, as we don't include 3rd party cookies (such as Google Analytics) within Kolibri, and all the cookies we do use are for "strictly necessary" purposes. So we don't need a bunch of checkbox options, etc.

**Comment by rtibbles** [comment]
> all the cookies we do use are for "strictly necessary" purposes. 

I don't think this is true - our `visitor_id` cookie is a tracking cookie.

**Comment by jamalex** [comment]
Ah yes, still not 3rd party, but as the site would function (from the end-user's perspective) without that cookie, it's not necessary indeed! So I guess we at least need one checkbox for disabling that one as desired.

**Comment by rtibbles** [comment]
Right, and it needs to be persistently available in some form, as people need to be able to revoke their consent in the future as well. Do you have a sense of how obtrusive the persistent availability has to be?

**Comment by jamalex** [comment]
I believe CookieBot recommended putting the opt-out within the cookie notice/policy page. That's what we do here, for instance:
https://learningequality.org/cookies/ (linked from the footer)

**Comment by jtamiace** [comment]
Here are the existing designs for this. This hasn't been reviewed yet, so I'm looking for feedback. I'm not sure what behaviors should happen when someone clicks "privacy policy" and whether we have copy for that.

![cookie.jpg](https://images.zenhubusercontent.com/6047f63852483ffd3d7d34a1/60f812cd-e1a8-4bfe-848e-c6200254e563)

Also see this related thread about tracking who and how many people are giving their consent 
https://learningequality.slack.com/archives/CN9S5A5EG/p1643749265330059?thread_ts=1643205211.013900&cid=CN9S5A5EG
